modules/security-context-constraints-psa-opting.adoc

```diff
@@ -10,22 +10,24 @@ You can enable or disable automatic pod security admission synchronization for m
 
 [IMPORTANT]
 ====
-Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
+Most namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
 
 * `default`
 * `kube-node-lease`
 * `kube-system`
 * `kube-public`
 * `openshift`
-* All system-created namespaces that are prefixed with `openshift-`, except for `openshift-operators`
+* System-created namespaces that are prefixed with `openshift-` and do not contain an Operator
 
-By default, all namespaces that have an `openshift-` prefix are not synchronized. You can enable synchronization for any user-created [x-]`openshift-*` namespaces. You cannot enable synchronization for any system-created [x-]`openshift-*` namespaces, except for `openshift-operators`.
+By default, namespaces that have an `openshift-` prefix with an Operator have pod security admission synchronization enabled. After a cluster service version (CSV) is created in an `openshift-*` namespace, pod security admission synchronization is turned on. The label inherits the permissions set in service account of the namespace.
+
+You can enable synchronization for any user-created [x-]`openshift-*` namespaces. You cannot enable synchronization for any system-created [x-]`openshift-*` namespaces, except for `openshift-operators`.
```

Comment:
```suggestion
You can enable synchronization for any user-created [x-]`openshift-*` namespaces. You cannot enable synchronization for any system-created [x-]`openshift-*` namespaces, except for `openshift-operators`.
```

A few questions:

- This is an artifact from the 4.11 docs. Is this still accurate?
- In OCP 4.11 customer-created `openshift-*` namespaces and system-created `openshift-*` namespaces were handled differently. Is that still the case in OCP 4.12?
- Is the `openshift-operators` namespace still handled differently? If so, how does it need to be documented?
- Does the synchronization occur for system-created `openshift-*` namespaces if an Operator is installed?

Reply:
> To put it another way, if an Operator is installed on a system-created openshift- * namespace, will PSA label synchronization occur?

no, it will not.  See: https://github.com/openshift/cluster-policy-controller/blob/523e4104074f88047de7d725d5c99ca4dbf8c167/pkg/psalabelsyncer/nsexemptions/nsexemptions.go#L9

those NSes will *never* have label syncing performed on them.